packages/botonic-react/src/webchat/components/emoji-picker.jsx

```diff
@@ -12,19 +12,29 @@ export const EmojiPicker = props => (
   </IconContainer>
 )
 
+const Background = styled.div`
+  width: 100%;
+  height: ${props => props.height}px;
+  position: absolute;
+  right: 0;
+  bottom: 0;
+`
+
 const Container = styled.div`
-  width: ${props => props.width || '100%'};
-  max-width: 400px;
   display: flex;
   justify-content: flex-end;
   position: absolute;
-  right: 0px;
-  top: -332px;
+  right: 3px;
+  top: -324px;
 `
+
 export const OpenedEmojiPicker = props => {
   return (
-    <Container width={props.width}>
-      <EmojiPickerComponent onEmojiClick={props.onClick} />
-    </Container>
+    <>
+      <Background onClick={props.onClick} height={props.height}></Background>
```

Comment:
I think this is the more "handy" solution at this moment, but I also think that this is probably a recurrent problem in react. I have done some little research and I have seen that maybe we can have a `hook` to do this, at the same time it can be reused in the future: https://stackoverflow.com/questions/54560790/detect-click-outside-react-component-using-hooks. If it's possible, I will try to invest some little time taking a look on it, and if you consider this appropiate, let's create a task to try to apply this.

Reply:
Agree!